### ISSUE TITLE
Add guidelines for social media

### ISSUE THREAD

**Issue by pllim** [astropy/astropy-project#449]
You can adapt from the defunct Twitter guideline removed in https://github.com/astropy/astropy-project/pull/444

**Comment by kelle** [comment]
I don't think we need a dedicated issue for this. It's kinda obvious that this would be under the purview of a Community Manager and that role is currently unfilled.
